config/_default/menus/main.en.yaml

```diff
@@ -3899,11 +3899,16 @@ menu:
       parent: llm_obs
       identifier: llm_obs_submit_evaluations
       weight: 5
+    - name: Cluster Map
```

Comment:
Instead of leaving this page at the top level of the nav bar, can we group this under something else, similar to how APM does with "Trace Explorer"?

Reply:
@barieom we had discussed adding a Trace Explorer section in the [LLM Observability doc set](https://docs.google.com/document/d/1XZQ4zs3woFuYR3S8BJnJjVKByvqqWdGwlDJgIaeq8N4/edit#heading=h.nbm2853yako4), but we haven't drafted any content for that yet. I was planning on doing that next, so we can hold off on this change for now, and I'll start drafting some content for the Trace Explorer?